### PR TITLE
Minor Go refinements

### PR DESCRIPTION
**Pull request by chadell**

### CODE REVIEW

File: tcpserver/tcpserver.go [hunk 1]
```diff
@@ -1,75 +1,74 @@
...
+	"log"
+	"net"
+	"strconv"
+	"time"
 )
 
-type Handler struct {
-    conn   net.Conn
-    closed chan bool
+type Handler struct { // Handler : describe what this function does
```
Comment (dcaba) [inline tcpserver/tcpserver.go]: I think this comment does not apply

File: tcpserver/tcpserver.go [hunk 2]
```diff
@@ -1,75 +1,74 @@
...
+			}
+			h.closed <- true // send to dispatcher, that connection is closed
+			return
+		}
+	}
 }
 
-type Dispatcher struct {
-    Handlers map[string]*Handler //`type:"map[ip]*Handler"`
+type Dispatcher struct { // Dispatcher : describe what this function does
```
Comment (dcaba) [inline tcpserver/tcpserver.go]: again

File: tcpserver/tcpserver.go [hunk 3]
```diff
@@ -1,75 +1,74 @@
...
+	Handlers map[string]*Handler //`type:"map[ip]*Handler"`
 }
 
-func (d *Dispatcher) AddHandler(conn net.Conn) {
-    addr := conn.RemoteAddr().String()
-    handler := &Handler{conn, make(chan bool, 1)}
-    d.Handlers[addr] = handler
+func (d *Dispatcher) AddHandler(conn net.Conn) { // AddHandler : describe what this function does
```
Comment (dcaba) [inline tcpserver/tcpserver.go]: again

File: tcpserver/tcpserver.go (hunk 1 above)
Comment (chadell) [reply tcpserver/tcpserver.go]: 👍 , I will add a proper description (I forgot it)

**Comment by chadell** [comment]
@dcaba , plz, do a review again (I rebased)

**Pull request merged**